interfaces/security_test.go

```diff
@@ -33,24 +34,34 @@ type SecuritySuite struct {
 	slot *Slot
 }
 
-var _ = Suite(&SecuritySuite{
-	plug: &Plug{
-		Snap:      "producer",
-		Name:      "plug",
-		Interface: "interface",
-		Apps:      []string{"hook"},
-	},
-	slot: &Slot{
-		Snap:      "consumer",
-		Name:      "slot",
-		Interface: "interface",
-		Apps:      []string{"app"},
-	},
-})
+var _ = Suite(&SecuritySuite{})
 
 func (s *SecuritySuite) SetUpTest(c *C) {
 	s.BaseTest.SetUpTest(c)
 	s.repo = NewRepository()
+	// NOTE: the names producer/consumer are confusing. They will be fixed shortly.
```

Comment:
I feel they are backwards now. Plug is the "consumer side" and slot is the "provider side" but the names here are the other way around.